linkinator.config.json

```diff
@@ -2,6 +2,7 @@
   "recurse": true,
   "skip": [
     "https://codecov.io/gh/googleapis/",
-    "www.googleapis.com"
+    "www.googleapis.com",
+    "https://github.com/googleapis/nodejs-logging/blob/master/samples/http-request.js"
```

Comment:
this is an argument for not generating README.md until the nightly run (although I did find an issue with synthtool generating these READMEs https://github.com/googleapis/synthtool/pull/336).

Reply:
If you have any ideas on how to work around this with linkinator, I'm all ears.  Haven't been able to figure this one out yet :)

Reply:
I think we should potentially land new samples first, and then let the nightly build add the new links... This only bites us when creating brand new libraries.